web-ide/proxy/src/config.json

```diff
@@ -2,7 +2,8 @@
     "devMode" : false,
     "http" : {
         "port" : 3000,
-        "managementPort" : 3001
+        "managementPort" : 3001,
+        "httpToHttpsPort" : 3002
```

Comment:
odd but the gcp loadbalancer being put in place does not have good redirect support, so I was told to open up another port for redirection. The load balancer will direct https traffic to our application by http:80 and direct http traffic to http:3002 so that the application can redirect it to https.

Reply:
Good comment. Can we get it in the code?

Reply:
can't comment json - it's partially documented in httpToHttps.ts but I'll add to that